tests/test_tradingcalendar.py

```diff
@@ -26,7 +26,6 @@
 
 class TestTradingCalendar(TestCase):
 
-    @nottest
```

Comment:
I'm not sure why it was disable but the test for BMF, TSE and LSE are also commented out but I'm pretty sure all but TSE will fail.